components/onboard-wizard/WizardStep.tsx

```diff
@@ -25,7 +26,18 @@ export default function WizardStep({ step }: { step: Step }) {
   } = useWizardController();
 
   const t = useTranslations('Generic');
-  const position = getElementPosition(targetElementId);
+  const [position, setPosition] = useState(getElementPosition(targetElementId));
+
+  useEffect(() => {
+    const updatePosition = () => {
+      setPosition(getElementPosition(targetElementId));
+    };
+
+    updatePosition();
+    window.addEventListener('resize', updatePosition);
+
+    return () => window.removeEventListener('resize', updatePosition);
+  }, [targetElementId]);
```

Comment:
A better way to do this is to make `getElementPosition` into a hook (or to create a new hook that uses the existing function), and then to put the reactivity there.